source/SIL.AppBuilder.Portal.Backend.Tests/Acceptance/Support/TestCUrrentUserContext.cs

```diff
@@ -0,0 +1,17 @@
+﻿using OptimaJet.DWKit.StarterApplication.Services;
```

Comment:
The filename is TestCUrrentUserContext according to the display here.  Minor but CU should be Cu.

Reply:
most def. been havin Visual Studio issues with renaming today